src/main/kotlin/racingcar/domain/AwardsStrategy.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+enum class AwardsStrategy(
+    val awards: (RacingGame) -> List<Car>
+) {
+
+    FIND_WINNERS( { cars ->
+        require(cars.value.isNotEmpty())
+
+        cars.value.groupBy { car ->
+            car.position.value
+        }.maxByOrNull { carsEntry ->
+            carsEntry.value.maxOf { car ->
+                car.position.value
+            }
+        }?.value!!
```

Comment:
```suggestion
        }?.value ?: emptyList()
```
non-null assertion(!!)은 가급적이면 사용을 지양하는 것이 좋습니다.
무조건 null이 아닌 경우라면 사용해도 좋으나, 프로그램에는 예외가 반드시 존재합니다.
특히 List와 같은 컬렉션은 빈 컬렉션을 반환하게 처리하면 좋습니다.